cmd/jsonnet.cpp

```diff
@@ -707,17 +708,11 @@ int main(int argc, const char **argv)
         const char *jsonnet_path_env = getenv("JSONNET_PATH");
         if (jsonnet_path_env != nullptr) {
             std::list<std::string> jpath;
-            char *input = strdup(jsonnet_path_env);
-            do {
-                const char *path = strtok(input, const_cast<char*>(PATH_SEP));
-                input = nullptr;
-                if (path != nullptr) {
-                    jpath.push_front(path);
-                } else {
-                    break;
-                }
-            } while (true);
-            free(input);
+            std::stringstream ss(jsonnet_path_env);
+            std::string path;
+            while (std::getline(ss, path, PATH_SEP)) {
+                jpath.push_front(path);
+            }
             for (const std::string &path : jpath) {
```

Comment:
Fold the two loops into one for good measure.

Reply:
It's like that to reverse the order.